docs/sprint-planning.md

```diff
@@ -0,0 +1,24 @@
+---
+id: sprint-planning
+title: Sprint Planning
+slug: /sprint-planning
+---
+
+1. Go to the "Stories" page in Shortcut
+1. Go to the "Needs Iteration" Space
+1. Assign the current Iteration to any Story that does not have an Interation and is in progress
+1. Go to the "Needs Points" Space
+1. Assign Points to any Stories that are unestimated
+1. Go to the "Everything" Space
+1. Uncheck the "No Iteration" Filter
+1. Go through each Column starting with "Acceptance", discuss the status of each Story, and take action if needed
```

Comment:
does this mean "go through each column from right-to-left"?

Reply:
Yes, do you feel that needs clariication?

Reply:
i think so, maybe
> Go through each Column from right to left (starting with "Acceptance") ...